subprojects/platform-jvm/src/main/java/org/gradle/platform/Architecture.java

```diff
@@ -25,5 +25,5 @@
  */
 @Incubating
 public enum Architecture {
-    I386, AMD64, AARCH64
+    X86, X86_64, AARCH64
```

Comment:
It might be good to have some documentation per enum value, explaining what they represent. As we expect plugin implementors to have to somewhat translate them.